src/polycubed/src/rest_server.cpp

```diff
@@ -26,12 +27,14 @@
 #include "polycube/services/response.h"
 #include "server/Resources/Data/AbstractFactory.h"
 #include "server/Server/ResponseGenerator.h"
+#include "config.h"
 
 namespace polycube {
 namespace polycubed {
 
 std::string RestServer::whitelist_cert_path;
 std::string RestServer::blacklist_cert_path;
+bool RestServer::startup;
```

Comment:
Add a comment or rename the variable. It does not sound obvious to me what a boolean "startup" means.

Reply:
Please add the following comment in the code:
// This boolean is true when the rest server starts. This will be used not to save the current topology at startup.
// This flag is then reset to false, and all the incoming configuration commands will be saved in the dump file.